experimenter/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormFeatureValue/validators.ts

```diff
@@ -347,26 +348,32 @@ export function schemaAutocomplete(schema: Record<string, unknown>) {
   };
 }
 
-/**
- * Parse from FML errors to Diagnostics
- */
-function parseFmlErrors(view: TestableEditorView): Diagnostic[] {
-  const diagnostics: Diagnostic[] = [];
-
-  const doc = view.state.doc;
-  const text = doc.toString();
-  const from = text.indexOf("error");
-  if (from === -1) return diagnostics;
-
-  const to = from + "error".length;
-
-  diagnostics.push(createDiagnostic(from, to, "oh no!"));
-  return diagnostics;
-}
+export function fmlLinter(
+  slug: string,
+  feature: getConfig_nimbusConfig_allFeatureConfigs,
+) {
+  return async function (view: TestableEditorView) {
+    const url = `/api/v5/fml-errors/${slug}/`;
+    const response = await fetch(url, {
+      method: "PUT",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        featureSlug: feature.slug,
+        featureValue: view.state.doc.toString(),
+      }),
+    });
+    const errors: Record<string, any>[] = await response.json();
+
+    return errors.map((e: Record<string, any>) => {
+      const position = documentPosition(
+        view.state.doc,
+        e["line"] + 1,
+        e["col"],
```

Comment:
is this 1-based or 0-based

Reply:
Yeah so the FML returns it 0 based and the code mirror piece expects 1 based, so I figured it made more sense to convert it here so it was clearer that code mirror expects it 1 based?  What do you think?

Reply:
Yeah +1 for doing it here.